environments/skeleton/{{cookiecutter.environment}}/tofu/variables.tf

```diff
@@ -29,9 +29,27 @@ variable "control_node_flavor" {
     description = "Flavor name for control name"
 }
 
-variable "login_nodes" {
-  type = map
-  description = "Mapping defining login nodes: key -> (str) nodename suffix, value -> (str) flavor name"
+variable "login" {
+  type = any
+  description = <<-EOF
+    Mapping defining groups of login nodes. Keys are names of groups.
```

Comment:
Ah, this is where the docs are :facepalm: 

Maybe the readme could describe _why_ you'd want multiple login groups?

Reply:
You might. One client has real login nodes, plus an ondemand node which is a different size. I agree its a bit nasty though.

Reply:
That makes complete sense, now you mention it.

Do the group names for the compute and login groups get exposed back up to ansible?

Reply:
Yeah. The compute groups here get turned into "{{ cluster_name }}_{{group_name}}" groups which are what is used for "stackhpc.openhpc" partition definition.